pattern/aws-dynamodb-kinesisstreams-s3/test/index.test.ts

```diff
@@ -1,15 +1,15 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: MIT-0
 
-import {expect as expectCDK, haveResource, ResourcePart} from '@aws-cdk/assert';
-import {AwsDynamoDBKinesisStreamsS3} from '../lib';
```

Comment:
Once NPM library for aws-dynamodb-kinesisstreams-s3 is upgraded with this change, We will make the change for this import as well.